Backend/User/src/main/java/user/service/ProfileService.java

```diff
@@ -185,10 +185,15 @@ public Map<String, String> getNamesByUUIDList(List<String> uuids) {
     @Transactional
     public void updateElo(String uuid, Integer elo) {
         ValidationUtil.validateUUID(uuid);
+
         if (elo == null) {
             throw new IllegalArgumentException("ELO is required");
```

Comment:
suggestion: could make it more so to force a boolean value check on elo (covering null, 0, false datatypes?) with [.valueOf ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean/valueOf) method and wrap the whole with `IllegalArgumentException("Invalid or Missing ELO")`